site/content/docs/5.0/components/alerts.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
This newline is redundant

Reply:
@XhmikosR Didn't notice that white space. 🤭
Updated again.